threading/channels.nim

```diff
@@ -286,19 +288,23 @@ template trySend*[T](c: Chan[T], src: T): bool =
   trySend(c, isolate(src))
 
 proc tryRecv*[T](c: Chan[T], dst: var T): bool {.inline.} =
-  ## Tries to receive a message from the channel `c` and fill `dst` with its value.
-  ## This returns immediately even if no message is found. Doesn't block.
+  ## Tries to receive an item from the channel `c` and fill `dst` with its value.
   ## 
-  ## This can fail for all sort of reasons, including a lack of messages in the channel
-  ## to receive or contention.
+  ## Returns immediately once the lock on the Channel is acquired and an attempt
+  ## at fetching an item from it was made. 
+  ## The proc will not wait for items to appear.
+  ## 
+  ## This will fail if the channel is empty.
   ##
   ## If it fails it returns `false`. Otherwise it returns `true`.
   channelReceive(c.d, dst.addr, sizeof(T), false)
 
 proc send*[T](c: Chan[T], src: sink Isolated[T]) {.inline.} =
-  ## Sends item to the channel. 
+  ## Sends item `src` to the channel `c`. 
   ## This blocks the sending thread until the item was successfully sent.
```

Comment:
```suggestion
  ## This blocks the sending thread until an item was successfully sent.
```

Reply:
Why "an" item in this case? This will block until `src` was sent, not just any message, right?

Reply:
It's just for consistency. Everywhere else the `src`/`dst` args are called with an indefinite article, probably meaning we don't care what an item really is. I'm not English native, so I don't have a strong opinion here and can be wrong so I don't mind either way, as long as it's consistent.

Reply:
Did it more uniformly. Used specific article "the" for sending / src and unspecific article "a" for receiving / dist, since you know the message you send but you do not know the message you receive, thus the difference makes sense there.